src/string_id_null_ids.cpp

```diff
@@ -45,3 +45,4 @@ MAKE_NULL_ID2( species_type, "spec_null" );
 MAKE_NULL_ID2( mutation_branch );
 MAKE_NULL_ID2( requirement_data, "null" );
 MAKE_NULL_ID2( body_part_struct, "NUM_BP" );
+MAKE_NULL_ID2( bionic_data, "", 0 );
```

Comment:
You don't need the `0` here, it's to set the null id for the matching `int_id`, but we don't have `int_id<bionic_data>`.